bin/xbps-query/xbps-query.1

```diff
@@ -88,6 +88,49 @@ If the first character is not '/' then it's a relative path of
 .Ar rootdir .
 .It Fl d, Fl -debug
 Enables extra debugging shown to stderr.
+.It Fl F, Fl -format Ar format
+Format for list output.
+.Bd -literal
```

Comment:
nit: this is a giant block, maybe it make more sense to do
```roff
Format for list output. See
.Sx FORMAT STRINGS
for syntax.
```
and make it a section of the page